packages/components/src/forms/Input/Input.android.tsx

```
@@ -1,13 +1,76 @@
+  if (extras.props.multiline || extras.props.numberOfLines > 1) {
+    return textAreaSizeVariant(val, extras);
+  }
+  const buttonStyles = getButtonSized(val, extras);
+  const paddingHorizontal = getSpace(val, {
+    shift: -1,
+    bounds: [2],
+  });
+  const fontStyle = getFontSized(val as any, extras);
+  // lineHeight messes up input on native
+  if (!isWeb && fontStyle) {
+    delete fontStyle.lineHeight;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)